specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/preview/2023-05-01-preview/migrate.json

```diff
@@ -11731,6 +11753,11 @@
           "$ref": "#/definitions/CloudSuitability",
           "description": "Gets a value indicating whether machine is suitable for the cloud platform\nselected.",
           "readOnly": true
+        },
+        "avsTargetNode": {
```

Comment:
Do we need to have avs prefix considering the Object itself has Avs in its name ?

Reply:
This has been verified from PM and wanted the same name in response. The target will always be avsNode. That is the reason for including this.